pkg/resourcemanager/mysql/mysqlhelper.go

```diff
@@ -0,0 +1,164 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"github.com/Azure/azure-service-operator/pkg/errhelp"
+	"github.com/Azure/azure-service-operator/pkg/helpers"
+	"github.com/Azure/azure-service-operator/pkg/resourcemanager/config"
+	"github.com/Azure/azure-service-operator/pkg/resourcemanager/iam"
+)
+
+// MSqlServerPort is the default server port for sql server
+const MySQLServerPort = 3306
+
+// MDriverName is driver name for psqldb connection
+const MySQLDriverName = "mysql"
+
+func GetMySQLDatabaseDNSSuffix() string {
+	result := "mysql.database.azure.com"
+
+	// TODO: I am pretty sure that the SQLDatabaseDNSSuffix is for Azure SQL and not MySQL...
```

Comment:
Is this comment vestigial? Presumably this works. :)

Reply:
I'm 99% sure this doesn't work. I've changed this implementation so that's cleared, left a comment, and filed a bug for the Go SDK to include a variable for this.